src/frontend.md

```diff
@@ -221,6 +221,24 @@ export function fakeTicket() {}
 
 1. If some color is repeated several times in code, extract it to theme.
 
+## Disposition of components
+
+- Prefer using local components `(host|mf)/src/components` with a few exceptions
+- Use UI Kit distribution if a component represents a design token (1) [websome](<https://www.figma.com/file/I8WglqziPB5pYlFIqF43JE/Client-UI-Kit-(Web)?node-id=0-1&t=MuFhhOjUXP4sFRr2-0>) or matches with [MUI](https://mui.com/material-ui/)
```

Comment:
what does mean `represents a design token`? do you mean smth like `a component is a part of design guide`?

`or matches with [MUI]` – could you explain how we correlate with MUI?

and why is `agent` ignored here?

Reply:
Updated. Take a look, please